templates/repo/issue/sidebar/development.tmpl

```diff
@@ -0,0 +1,105 @@
+{{if not .Issue.IsPull}}
+	<div class="divider"></div>
+
+	<span class="text"><strong>{{ctx.Locale.Tr "repo.issues.development"}}</strong></span>
+	<div class="ui devlinks list">
+	{{/* AllowedRepos not empty means login user can create new branch in some of the repositories */}}
+	{{if .ShowCreateBranchLink}}
+		<div class="tw-items-center">
+			<a class="tw-mt-1 fluid ui show-modal" data-modal="#create_branch">{{ctx.Locale.Tr "repo.branch.new_branch"}}</a>
+		</div>
+	{{end}}
+	{{range .DevLinks}}
```

Comment:
Please don't use the data from root template data.

Most side bar parts have been refactored to use their own "struct" types to make code stable.